tests/external_executor/external_executor.py

```diff
@@ -137,132 +117,21 @@ def test_executor_registration(network, args):
             ) as channel:
                 should_pass = node == primary and credentials == executor_credentials
                 try:
-                    rd = Service.KVStub(channel).StartTx(Empty())
-                    assert should_pass, "Expected StartTx to fail"
-                    assert not rd.HasField("optional")
+                    stub = Service.KVStub(channel)
+                    for m in stub.Activate(Empty(), timeout=1):
+                        assert m.HasField("activated"), f"Expected only an activated message, not: {m}"
                 except grpc.RpcError as e:
-                    # NB: This failure will have printed errors like:
-                    # These are harmless and expected, and I haven't found a way to swallow them
-                    assert not should_pass
-                    # pylint: disable=no-member
-                    assert e.code() == grpc.StatusCode.UNAUTHENTICATED, e
-
-    return network
-
-
-@reqs.description("Test basic KV operations via external executor app")
-def test_kv(network, args):
```

Comment:
Have deleted this test entirely. It was helpful for early prototyping, but I don't think its useful anymore. The required functionality is tested by fuller tests, and the shims necessary to test this (essentially creating, registering, and activating a fake Executor so we can query its auth'd API) were getting increasingly, and unnecessarily, complex.